R/loocv.R

```diff
@@ -191,8 +191,14 @@ is.baggr_cv <- function(x) {
 #' Given multiple [loocv] outputs, calculate differences in their expected log
 #' predictive density.
 #'
-#' @param x An object of class `baggr_cv` or a list of such objects.
+#' @param x A list of `baggr_cv` objects, with a minimum of 2 objects required for comparison.
+#'          `baggr_cv` objects can be created via the `loocv` method. In instances where a list
```

Comment:
You want to link to documentaiton, `[loocv]` not `loocv` (check Roxygen tutorials)

Reply:
Got it! I'll change this.